modules/core/src/main/scala/ch/epfl/bluebrain/nexus/admin/core/resources/Resources.scala

```diff
@@ -62,37 +66,37 @@ class Resources[F[_], A: IdResolvable: TypeFilterExpr](agg: Agg[F], sparqlClient
     * @return () or the appropriate rejection in the ''F'' context
     */
   @silent
-  def validateUpdate(id: A, value: Json): F[Unit] = F.pure(())
+  def validateUpdate(id: A, value: Json): F[Unit]
+
+  def tags: Set[String]
```

Comment:
If you put the tags at that level and you remove them from every event, you won't be able to tag events independently, meaning if we want to have specific tags depending on the payload (type or whatever) we can't.

Reply:
Yes, but for now we only have static tags based on resource type, we can change it when we'll need it, but I don't think we'll need that in `nexus-admin`.